react/training/src/redux/login/reducers.js

```diff
@@ -0,0 +1,17 @@
+import { actionTypes } from './actions';
+
+const initialState = {
+  userAuthenticated: false
+};
+
+export default function logInReducer(state = initialState, action) {
+  switch (action.type) {
```

Comment:
Here when you need to get info from an `API` you need to divide your actions like this: 
```
  switch (action.type) {
    case actionTypes.LOGIN_REQUEST:
      return {
        ...state,
        ...,
      };
    case actionTypes.LOGIN_SUCCESS:
      return {
        ...state,
        ...,
        userAuthenticated: action.payload
      };
    case actionTypes.LOGIN_FAILURE:
      return { ...state, loginError: true, userAuthenticated: false };
    default:
      return state;
```